File: vb/RevisionQuestion13.vb
Module RevisionQuestion13
	Sub Main()
		Dim price, sum As Double
		Dim quantity As Integer

		Console.Write("Input price (input 0 to quit): ")
		price = Console.ReadLine()

		Console.Write("Input quantity (input 0 to quit): ")
		quantity = Console.ReadLine()

		Console.WriteLine()

		While price > 0.0 And quantity > 0
			sum += price * quantity

			Console.Write("Input price (input 0 to quit): ")
			price = Console.ReadLine()

			Console.Write("Input quantity (input 0 to quit): ")
			quantity = Console.ReadLine()

			Console.WriteLine()
		End While

		Console.WriteLine("Total sum to be paid: ${0}", sum)
		Console.ReadKey()
	End Sub
End Module

File: py/RevisionQuestion13.py
def get_price_quantity():
    price = float(input("Input price (input 0 to quit): "))
    quantity = int(input("Input quantity (input 0 to quit): "))
    return (price, quantity)


sum = 0.0

price_quantity = get_price_quantity()

while price_quantity[0] > 0.0 and price_quantity[1] > 0:
    sum += price_quantity[0] * price_quantity[1]

    price_quantity = get_price_quantity()

print("Total sum to be paid: ${0}".format(sum))
